core/src/main/java/com/epam/esm/mapping/CertificateRowMapper.java

```diff
@@ -0,0 +1,44 @@
+package com.epam.esm.mapping;
+
+import com.epam.esm.entities.Certificate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Component;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.time.LocalDateTime;
+
+/**
+ * Maps {@link Certificate} from current row in {@link ResultSet}
+ */
+@Component
+public class CertificateRowMapper implements RowMapper<Certificate> {
+
+    private static final String ID = "id";
+    private static final String NAME = "name";
+    private static final String DESCRIPTION = "description";
+    private static final String PRICE = "price";
+    private static final String DURATION = "duration";
+    private static final String CREATE_DATE = "create_date";
+    private static final String LAST_UPDATE_DATE = "last_update_date";
+
+    @Override
+    public Certificate mapRow(ResultSet resultSet, int rowNum) throws SQLException {
```

Comment:
@KukharchukDaniil For this action you can use MapStruct or ModelMapper libraries. http://modelmapper.org/    https://mapstruct.org/